msal/src/main/java/com/microsoft/identity/nativeauth/utils/ParcelExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.identity.nativeauth.utils
+
+import android.os.Build
+import android.os.Parcel
+import java.io.Serializable
+
+/**
+ * Helper method to deal with Parcel.readSerializable being deprecated in newer Android versions.
```

Comment:
I'm personally curious about this file. Is it used to suppress warnings? Feel free to resolve the comment.

Reply:
No, it's to accurately deal with warnings. Warnings are there to indicate that something shouldn't be used anymore, so this method tackles that. Ignoring warnings is another way to get rid of them, but not the right way (in this case).